docs/source/markdown/podman-container-checkpoint.1.md

```diff
@@ -52,12 +52,24 @@ exported to a tar.gz file it is possible with the help of **--ignore-rootfs**
 to explicitly disable including changes to the root file-system into
 the checkpoint archive file.
 
+#### **--pre-checkpoint**, **-P**
+
+Dump container's memory information only, leave the container running after this. Later
+operations will override the previous ones. It means a contaniner only has one pre-dump images.
+
+#### **--with-previous**
+
+This only works without **--pre-checkpoint**. It will checkout the container with previous criu image files in pre-dump.
```

Comment:
I'm not clear what criu is or how this operates.  First attempt, may adjust later.
```suggestion
Check out the container with previous criu image files in pre-dump.

```